index.js

```diff
@@ -180,72 +180,89 @@
   //  Value :: Location
   var Value = 'Value';
 
-  //  _funcPath :: (Boolean, Array String, a) -> Nullable Function
-  function _funcPath(allowInheritedProps, path, _x) {
-    var x = _x;
-    for (var idx = 0; idx < path.length; idx += 1) {
-      var k = path[idx];
-      if (x == null || !(allowInheritedProps || has(k, x))) return null;
-      x = x[k];
-    }
-    return typeof x === 'function' ? x : null;
-  }
+  //  prefix :: String
+  var prefix = 'fantasy-land/';
 
-  //  funcPath :: (Array String, a) -> Nullable Function
-  function funcPath(path, x) {
-    return _funcPath(true, path, x);
+  //  funcProp :: (String, a) -> Nullable Function
+  function funcProp(prop, value) {
+    var f = value && value[prop];
```

Comment:
Please use `value != null` as the guard. I like to avoid type coercion. ;)

This might be clearer:

```javascript
return value != null && typeof value[prop] === 'function' ? value[prop] : null;
```

`s/value/x/g` would allow it to fit on a single line. :)